todo/services/team_service.py

```
@@ -0,0 +1,85 @@
+                description=created_team.description,
+                poc_id=str(created_team.poc_id) if created_team.poc_id else None,
+                created_by=str(created_team.created_by),
+                updated_by=str(created_team.updated_by),
+                created_at=created_team.created_at,
+                updated_at=created_team.updated_at,
+            )
+
+            return CreateTeamResponse(team=team_dto)
+
+        except Exception as e:
+            raise ValueError(str(e)) 
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)